arangod/Network/Methods.h

```diff
@@ -96,6 +96,14 @@ FutureRes sendRequest(ConnectionPool* pool, DestinationId destination,
                       RequestOptions const& options = {},
                       Headers headers = {});
 
+/// @brief send a request to all responsible servers for a given shard; use this
+/// method to send a message to followers as well as leader
+std::vector<FutureRes> sendRequests(ConnectionPool* pool, DestinationId destination,
```

Comment:
seems like this should not be here ? since it really just deals with shards the behaviour is also not that related to the rest of the sendRequest methods

Reply:
I suppose. I mean ... the purpose is to do exactly what `sendRequest` does, just with multiple targets. We could potentially unpack this so that it receives a vector of resolved destinations, and then use that as the internal basis for `sendRequest` (with a single vector entry). It just seems odd to me to reproduce the `sendRequest` functionality outside the `network` space just to get an overload on the destination resolution behavior.

Reply:
but its not the same thing, it just works for `shard:` destinations